microsoft/testsuites/ltp/ltp.py

```diff
@@ -125,7 +125,7 @@ def run_test(
         self.run(parameters, sudo=True, force_run=True, timeout=12000)
 
         # to avoid no permission issue when copying back files
-        self.node.tools[Chmod].update_folder("/opt/ltp", "a+rwX", sudo=True)
+        self.node.tools[Chmod].update_folder("/opt", "a+rwX", sudo=True)
```

Comment:
It's not good to change permission of a system folder. How about creating the ltp folder first, and then change its permission? It may need to change following line to copy into that folder, instead of creating another layer of "ltp".

Reply:
Ok I will change the solution to avoid changing the permissions of a system folder and test that.

Reply:
Changed the log location to the working directory per a suggestion and that resolved the permissions issue.